app/src/main/java/com/arctouch/codechallenge/home/HomePresenter.kt

```diff
@@ -0,0 +1,46 @@
+package com.arctouch.codechallenge.home
+
+import android.view.View
+import com.arctouch.codechallenge.api.*
+import com.arctouch.codechallenge.data.Cache
+import com.arctouch.codechallenge.data.Repository
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+
+
+class HomePresenter(private val view: HomeInterface.View){
+
+
+    fun onCreateCalled(repository: Repository){
+        val client = Client()
+        val api = client.getClient().create(TmdbApi::class.java)
+        loadGenres(api, repository)
+        loadMovies(api, repository)
```

Comment:
Which of these two API calls will finish first? Does the order matter?

Reply:
Yes the one that ends first is the loadGenre, the order matters yes, if I reverse the order the one who will finish first is loadMovies.